docs/guides/release.md

```diff
@@ -0,0 +1,44 @@
+# release binaries
+
+## Table of Contents
+* [Introduction](#intorduction)
+* [Generate](#generate)
+* [Add architecture](#add architecture)
+* [Remove architecture](#remove architecture)
+
+## Introduction
+In this file I will describe, how we can build the release binaries. 
```

Comment:
In this file I will describe, how we can build the release binaries  -->

This section describes how can we build the release binaries.